api/admin/controller.py

```diff
@@ -1071,26 +1072,26 @@ def admin_auth_services(self):
 
             return dict(
                 admin_auth_services=auth_services,
-                providers=ExternalIntegration.ADMIN_AUTH_PROVIDERS,
+                providers=ExternalIntegration.ADMIN_AUTH_PROTOCOLS,
             )
 
-        provider = flask.request.form.get("provider")
-        if not provider:
+        protocol = flask.request.form.get("provider")
```

Comment:
I didn't change `provider` to `protocol` in the web app because I think it makes sense to continue to call this a "provider" in user-facing applications. I can change it though if we want to be consistent.